allauthdemo/settings.py

```diff
@@ -0,0 +1,341 @@
+# -*- coding:utf-8 -*-
+import os
+
+import environ
+
+env = environ.Env(
+	DEBUG=(bool, True),
+	CACHE_URL=(str,  'locmemcache://'),
+	EMAIL_URL=('EMAIL_URL', 'smtp://[user]:[password]@[host.domain.tld]:[portnumber]'),
+)
+
+DEBUG = env('DEBUG')
+development=True
+SECRET_KEY = ''
```

Comment:
Add back the environment variable, causes an error on deploy.

Reply:
I have this working fine locally.